include/mpm_base.tcc

```diff
@@ -913,3 +912,28 @@ void mpm::MPMBase<Tdim>::particle_entity_sets(const Json& mesh_props,
                    exception.what());
   }
 }
+
+// Initialise Damping
+template <unsigned Tdim>
+bool mpm::MPMBase<Tdim>::initialise_damping(const Json& damping_props) {
+
+  // Read damping JSON object
+  bool status = true;
+  try {
+    // Read damping type
+    if (damping_props.find("type") != damping_props.end())
+      damping_type_ = damping_props["type"].template get<mpm::MPMBase<Tdim>::Damping>;
```

Comment:
Use `.at`

Reply:
Should I also remove this if as I am supposed to with the damping factor?

Reply:
Yes